plugins/modules/ecs_service.py

```diff
@@ -558,18 +582,20 @@ def is_matching_service(self, expected, existing):
 
         return True
 
-    def create_service(self, service_name, cluster_name, task_definition, load_balancers,
-                       desired_count, client_token, role, deployment_configuration,
+    def create_service(self, service_name, cluster_name, platform_version, task_definition, load_balancers,
+                       desired_count, client_token, role, deployment_controller, deployment_configuration,
                        placement_constraints, placement_strategy, health_check_grace_period_seconds,
                        network_configuration, service_registries, launch_type, scheduling_strategy):
 
         params = dict(
             cluster=cluster_name,
             serviceName=service_name,
+            platformVersion=platform_version,
```

Comment:
This is going to cause an exception when platform_version isn't set (it'll default to None, which Boto3 doesn't like.)